packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -0,0 +1,153 @@
+    .label-hidden {
+      clip: rect(0 0 0 0);
+      cursor: default;
+      visibility: hidden;
+    }
+
+    /* stylelint-disable selector-max-compound-selectors */
+    .label {
+      cursor: pointer;
+      display: flex;
+      font-size: var(--terra-form-checkbox-font-size, 1rem);
+      line-height: 1.42em;
+      padding-left: var(--terra-form-checkbox-padding-left, none);
+      position: relative;
+    }
+
+    /* Native checkbox */
+    .native-input {
+      border: var(--terra-form-checkbox-native-input-border, none);
+      clip: var(--terra-form-checkbox-native-input-clip, none);
+      cursor: var(--terra-form-checkbox-native-input-cursor, pointer);
+      font-size: var(--terra-form-checkbox-native-input-font-size, inherit);
+      height: var(--terra-form-checkbox-native-input-height, none);
+      margin-left: var(--terra-form-checkbox-native-input-margin-left, 0.071rem);
+      margin-right: var(--terra-form-checkbox-native-input-margin-right, 0.143rem);
+      overflow: var(--terra-form-checkbox-native-input-overflow, none);
+      padding: var(--terra-form-checkbox-native-input-padding, none);
+      position: absolute;
+      width: var(--terra-form-checkbox-native-input-width, none);
+    }
+
+    .inner-span {
+      margin-left: var(--terra-form-checkbox-inner-span-margin-left, 1.4em);
+    }
+
+    /* Custom checkbox */
+    .native-input + .inner-span::before {
+      background-color: var(--terra-form-checkbox-background-color, none);
+      background-image: var(--terra-form-checkbox-background-image, none);
+      background-position: center center;
+      background-repeat: no-repeat;
+      border-color: var(--terra-form-checkbox-border-color, none);
+      border-radius: var(--terra-form-checkbox-border-radius, none);
+      border-style: var(--terra-form-checkbox-border-style, none);
+      border-width: var(--terra-form-checkbox-border-width, none);
+      content: '';
+      cursor: pointer;
+      display: inline-block;
+      height: var(--terra-form-checkbox-height, none);
+      left: 0;
+      margin-top: var(--terra-form-checkbox-margin-top, none);
+      position: absolute;
+      top: 0;
+      visibility: visible;
+      width: var(--terra-form-checkbox-width, none);
+    }
+
+    /* Custom checkbox in checked state */
+    .native-input:checked + .inner-span::before {
+      background-color: var(--terra-form-checkbox-checked-background-color, none);
+      border-color: var(--terra-form-checkbox-checked-border-color, none);
+      border-radius: var(--terra-form-checkbox-checked-border-radius, none);
+      border-style: var(--terra-form-checkbox-checked-border-style, none);
+      border-width: var(--terra-form-checkbox-checked-border-width, none);
+      color: var(--terra-form-checkbox-symbol-color, none);
+      content: var(--terra-form-checkbox-symbol, none);
+      height: var(--terra-form-checkbox-checked-height, none);
+      left: 0;
+      line-height: 1em;
```

Comment:
Line-height of 1 is usually bad, you should check with @neilpfeiffer on this.

Reply:
^ to add to this, line-height should be unitless